src/App.jsx

```diff
@@ -14,70 +16,79 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [title, setTitle] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  const handleTitleChange = (good) => {
+    if (good === title
+      || !good
+      || typeof good !== 'string') {
+      setTitle('');
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+      return;
+    }
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+    setTitle(good);
+  };
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {title !== ''
+          ? `${title} is selected`
+          : 'No goods selected'
+        }
 
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
+        {title !== ''
+          && (
             <button
-              data-cy="RemoveButton"
+              data-cy="ClearButton"
               type="button"
-              className="button is-info"
-            >
-              -
-            </button>
-          </td>
+              className="delete ml-3"
+              onClick={handleTitleChange}
```

Comment:
I would recommend either creating a separate method for clearing or passing `() => handleTitleChange('')`